src/main/java/de/retest/recheck/cli/subcommands/Completion.java

```diff
@@ -0,0 +1,38 @@
+package de.retest.recheck.cli.subcommands;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import picocli.AutoComplete;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Spec;
+import picocli.CommandLine.Option;
+import picocli.CommandLine.Model.CommandSpec;
+
+@Command( name = "completion", description = "Generate and display auto completion script." )
+public class Completion implements Runnable {
+
+	private static final Logger logger = LoggerFactory.getLogger( Completion.class );
+
+	@Option( names = "--help", usageHelp = true, hidden = true )
+	private boolean displayHelp;
+
+	@Spec
+	protected CommandSpec spec;
+
+	@Override
+	public void run() {
+		CommandLine root = spec.commandLine();
+		while ( root.getParent() != null ) {
+			root = root.getParent();
+		}
+		final String script = AutoComplete.bash( root.getCommandName(), root );
```

Comment:
Does `bash` mean it is only Bash-specific? Or is it a file that includes both, Bash and ZSH?

Reply:
I think AutoComplete.bash includes both, bash and ZSH. AutoComplete also provides no special method for ZSH.